testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/DefaultGradleDistribution.groovy

```diff
@@ -343,7 +343,7 @@ class DefaultGradleDistribution implements GradleDistribution {
 
     @Override
     boolean isHasTestDisplayNames() {
-        return isSameOrNewer("8.8")
+        return isSameOrNewer("8.8-rc-1")
     }
```

Comment:
@ov7a I changed that this make sure all 8.8 versions are included.

Reply:
Thanks!